react/features/calendar-sync/functions.native.js

```diff
@@ -9,6 +13,37 @@ export * from './functions.any';
 
 const logger = require('jitsi-meet-logger').getLogger(__filename);
 
+/**
+ * Adds a Jitsi link to a calendar entry.
+ *
+ * @param {string} id - The ID of the calendar entry.
+ * @param {string} link - The link to add info with.
+ * @returns {Promise<*>}
+ */
+export function addLinkToCalendarEntry(id: string, link: string) {
+    return new Promise((resolve, reject) => {
+        const shareInfoText = getShareInfoText(link, false, true);
```

Comment:
This will conflict with Damyan's PR. He changed `getShareInfoText` to return a Promise and to currently be dispatched.

Reply:
I didn't know that, thanks for noticing!